.github/workflows/update-buildconfigs.yaml

```
@@ -0,0 +1,83 @@
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "GitHub Actions"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)